Libraries/Hotcakes.Commerce/Payment/Methods/PaypalExpress.cs

```diff
@@ -140,37 +133,16 @@ public bool Capture(Transaction t, HotcakesApplication app)
                             MessageType.Information));
                         return true;
                     }
-                    if (captureResponse.DoCaptureResponseDetails.PaymentInfo.PaymentStatus ==
-                        PaymentStatusCodeType.InProgress)
-                    {
-                        t.Result.Succeeded = true;
-                        t.Result.ResponseCode = "PENDING";
-                        t.Result.ResponseCodeDescription = "PayPal Express Payment IN PROGRESS";
-                        t.Result.Messages.Add(new Message("Paypal Express Payment PENDING. In Progress.", "OK",
-                            MessageType.Information));
-                        return true;
-                    }
-                    if (captureResponse.DoCaptureResponseDetails.PaymentInfo.PaymentStatus == PaymentStatusCodeType.None)
+                    if (captureResponse.Result<PayPalCheckoutSdk.Payments.Capture>().Status == "DENIED")
                     {
                         t.Result.Succeeded = true;
                         t.Result.ResponseCode = "PENDING";
-                        t.Result.ResponseCodeDescription = "PayPal Express Payment: No Status Yet";
-                        t.Result.Messages.Add(new Message("Paypal Express Payment PENDING. No Status Yet.", "OK",
+                        t.Result.ResponseCodeDescription = "PayPal Express Payment DENIED";
+                        t.Result.Messages.Add(new Message("Paypal Express Payment DENIED.", "OK",
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.